Libraries/JavaScriptAppEngine/Initialization/exceptionOccurHint.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule exceptionOccurHint
+ */
+
+'use strict';
+
+const SourceMapsCache = require('SourceMapsCache');
+const parseErrorStack = require('parseErrorStack');
+
+function exceptionOccurHint() {
+  const stack =  parseErrorStack(new Error());
+  const renderFrame = stack.find((frame) => frame.methodName.endsWith('render'));
```

Comment:
I have an awesome idea. We should show user line of code. For ```react-native``` apps it is easy - everything not inside  ```node_modules```. But for examples https://github.com/facebook/react-native/tree/master/Examples it is not so easy because ```react-native``` not under ```node_modules``` folder. Is there any way to get app root directory?